### PR TITLE
Lesson 3 add to storybook elements

### PR DESCRIPTION
**Pull request by castortroy06**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/** @type { import('@storybook/react-webpack5').StorybookConfig } */
```
Comment (NikitaUstsiushenka) [inline .storybook/main.js:1]: please remove extra comment

File: package.json [hunk 2]
```diff
@@ -10,19 +10,21 @@
...
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
   },
   "eslintConfig": {
```
Comment (NikitaUstsiushenka) [inline package.json]: ESLint config better move to the separate file .eslintrs.js

File: src/components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
```
Comment (NikitaUstsiushenka) [inline src/components/MovieDetails/MovieDetails.js]: Please change double quotes with single quotes

File: src/components/MovieDetails/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <img className="col-4" data-testid="image" alt={name} src={`/images/movies/${imageUrl}`} />
+            <div className="movie-details--container col-8">
+                <h2 data-testid="name" className="movie-details--name">{name}<span className="movie-details--rate">{rate}</span></h2>
+                <p data-testid="categories" className="movie-details--categories">{genres.join(', ')}</p>
+                <p className="movie-details--year-duration">
+                    <span data-testid="year" className="movie-details--year">{releaseYear}</span>
+                    <span data-testid="duration" className="movie-details--duration">{duration}</span>
+                </p>
+                <p data-testid="description" className="movie-details--description">{description}</p>
+
```
Comment (NikitaUstsiushenka) [inline src/components/MovieDetails/MovieDetails.js]: Please remove extra new line

File: src/components/MovieDetails/MovieDetails.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import MovieDetails from "./MovieDetails";
```
Comment (NikitaUstsiushenka) [inline src/components/MovieDetails/MovieDetails.stories.jsx]: Please change double quotes to single quotes

File: src/components/MovieDetails/__snapshots__/MovieDetails.test.js.snap [hunk 6]
```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (NikitaUstsiushenka) [inline src/components/MovieDetails/__snapshots__/MovieDetails.test.js.snap]: No need to commit snapshots

File: src/components/MovieTile/MovieTile.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
```
Comment (NikitaUstsiushenka) [inline src/components/MovieTile/MovieTile.js]: Please change double quotes to single quotes

File: src/components/MovieTile/MovieTile.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+import './movie-tile.scss';
+
+const MovieTile = ({ imageUrl, name, releaseYear, genres, onClick }) => {
```
Comment (NikitaUstsiushenka) [inline src/components/MovieTile/MovieTile.js:5]: Please implement props using PropTypes

File: src/components/MovieTile/__snapshots__/MovieTile.test.js.snap [hunk 9]
```diff
@@ -0,0 +1,69 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (NikitaUstsiushenka) [inline src/components/MovieTile/__snapshots__/MovieTile.test.js.snap]: No need to commit snapshots

File: src/components/SortControl/SortControl.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
```
Comment (NikitaUstsiushenka) [inline src/components/SortControl/SortControl.js]: Please change double quotes to single quotes

File: src/components/SortControl/SortControl.test.js [hunk 11]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+//import '@testing-library/jest-dom/extend-expect';
```
Comment (NikitaUstsiushenka) [inline src/components/SortControl/SortControl.test.js]: Please remove comment

File: src/components/SortControl/__snapshots__/SortControl.test.js.snap [hunk 12]
```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```
Comment (NikitaUstsiushenka) [inline src/components/SortControl/__snapshots__/SortControl.test.js.snap]: No need to commit snapshots

**Comment by castortroy06** [comment]
image
image
image

File: src/components/MovieDetails/MovieDetails.js [hunk 13]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './movie-details.scss';
+
+const MovieDetails = ({ imageUrl, name, releaseYear, duration, genres, description, rate, onClick }) => {
```
Comment (NikitaUstsiushenka) [inline src/components/MovieDetails/MovieDetails.js:4]: Apply PropTypes here

**Review by NikitaUstsiushenka — APPROVED** [review]

**Pull request merged**